src/Apps/Order/Components/CheckoutAddress.tsx

```diff
@@ -23,6 +35,18 @@ export type AddressChangeHandler = (
   key: keyof Address
 ) => void
 
+enum ErrorModalTitle {
+  general = "Check your delivery address",
+  suggested = "Confirm your delivery address",
+}
+
+enum AddressSuggestionRadioButton {
+  recommended = "Recommended",
+  user_address = "What you entered",
+}
+
+const MODAL_WIDTH = 549
```

Comment:
This is a strangely specific value? Where's that coming from?

Reply:
this comes from designs. I'll arrange to fit mobile.

Reply:
I just doubt `549` has some intentional design significance.

Reply:
yeah agreed